sweetercat/utils.py

```diff
@@ -63,6 +63,19 @@ def readSC():
     return df, plots
 
 
+def short_readSC(nrows=None):
```

Comment:
This is a good idea for testing. But how about just add `nrows=None` for the normal `readSC` function?